Cargo.toml

```diff
@@ -72,6 +72,7 @@ commander = { path = "util/commander" }
 suzuka-config = { path = "networks/suzuka/suzuka-config" }
 monza-config = { path = "networks/monza/monza-config" }
 # util
+flocks = { path = "util/flocks" }
 godfig = { path = "util/godfig" }
```

Comment:
Same with `godfig` its not immedietly apparent to the reader what this means. We could simply call it `system-config` or `dynamic-config`.

Reply:
`godfig` I think is branded. Right, @andyjsbell ?

Reply:
Also, I generally subscribe to the doctrine of using novel names when creating opinionated crates and similar.